apps/win32/react-native.config.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs');
```

Comment:
I didn't think win32 used the CLI bundling so I didn't think it needed a react-native.config.js file. (I could very well be wrong, since bundling is half magic anyways)

I would expect reactNativePath to be set to reference react-native-win32 however in this project.